src/interface/desktop/chat.html

```diff
@@ -689,7 +694,30 @@
 
                 fetch(url, { method: 'POST', body: formData, headers})
                     .then(response => response.ok ? response.json() : Promise.reject(response))
-                    .then(data => { chatInput.value += data.text; })
+                    .then(data => { chatInput.value += data.text.trimStart(); autoResize(); })
+                    .then(() => {
+                        // Don't auto-send empty messages
+                        if (chatInput.value.length === 0) return;
+
+                        // Send message after 3 seconds, unless stop send button is clicked
+                        sendButtonImg.style.display = 'none';
+                        stopSendButtonImg.style.display = 'initial';
+
+                        // Start the countdown timer UI
+                        document.getElementById('countdown-circle').style.animation = "countdown 3s linear 1 forwards";
```

Comment:
I think it would be neat if the desktop app worked in the model of:
1. Click record button
2. Speak
3. Click stop

And from there, auto send the recording. IMO holding down the record button isn't a common desktop-based design pattern.

Reply:
Great point! I've updated the UX to use 2-click to send audio message on Desktop (more specifically on devices with pointer devices like mouse)